dist/technology radar - complete.csv

```diff
@@ -1,7 +1,8 @@
 name,author,date,ring,quadrant,isNew,description,exposure,order
 .Net core,,1/11/2020,adopt,Languages & Frameworks,FALSE,".NET Core is a cross-platform .NET implementation for websites, servers, and console apps on Windows, Linux, and macOS.  <span class=""infoLink"">More info: <a href=""https://docs.microsoft.com/en-us/dotnet/core/introduction"" target=""_blank"">Click here</a></span>",all,10
 ".Net ",,1/11/2020,adopt,Languages & Frameworks,FALSE,".NET is an open source developer platform, created by Microsoft, for building many different types of applications. <span class=""infoLink"">More info: <a href=""https://dotnet.microsoft.com/learn/dotnet/what-is-dotnet"" target=""_blank"">Click here</a></span>",external,20
-Ruby,,1/11/2020,adopt,Languages & Frameworks,FALSE,"Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.  <span class=""infoLink"">More info: <a href=""https://www.ruby-lang.org/en/documentation/"" target=""_blank"">Click here</a></span>",external,30
+Ruby,,1/11/2020,adopt,Languages & Frameworks,FALSE,"Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.  <span class=""infoLink"">More info: <a href=""https://www.ruby-lang.org/en/documentation/"" target=""_blank"">Click here</a></span>",all,30
+Rails,,2/05/2021,adopt,Languages & Frameworks,FALSE,"Rails Elevator Pitch Ruby on Rails is an open-source web framework that's optimized for programmer happiness and sustainable productivity.  <span class=""infoLink"">More info: <a href=""https://rubyonrails.org//"" target=""_blank"">Click here</a></span>",all,31
```

Comment:
Is "Rails Elevator Pitch" meant to be included in the description of Rails?

Reply:
Dat was zeker niet de bedoeling, zijn die pull requests toch weer ergens goed voor. Fixed